frontend/src/pages/application/DeviceGroup/Settings/index.vue

```diff
@@ -25,6 +25,7 @@ export default {
         SectionSideMenu, SectionTopMenu
     },
     mixins: [permissionsMixin],
+    inheritAttrs: false,
```

Comment:
not related to the current scope, but prevents development warnings due to the component inheriting unspecified props from it's parent route. Passing props to vue routes is an antipattern